### PR TITLE
Translations

### PR DESCRIPTION
**Pull request by LeandroStanger**

### CODE REVIEW

File: com.anydesk.Anydesk.metainfo.xml [hunk 1]
```diff
@@ -3,15 +3,16 @@
   <id>com.anydesk.Anydesk.desktop</id>
   <name>Anydesk</name>
   <project_license>LicenseRef-proprietary</project_license>
-  <developer_name>Anydesk</developer_name>
-  <summary>Your desk at any desk</summary>
+  <developer_name>AnyDesk Software GmbH</developer_name>
   <metadata_license>CC0-1.0</metadata_license>
+  <summary>Connect to a computer remotely</summary>
+  <summary xml:lang="pt_BR">Conectar com um computador remotamente</summary>
+  <summary xml:lang="es">Conéctese a un ordenador de forma remota</summary>  <metadata_license>CC0-1.0</metadata_license>
```
Comment (wjt) [inline com.anydesk.Anydesk.metainfo.xml]:
```suggestion
  <summary xml:lang="es">Conéctese a un ordenador de forma remota</summary>
```

File: com.anydesk.Anydesk.metainfo.xml [hunk 2]
```diff
@@ -3,15 +3,16 @@
   <id>com.anydesk.Anydesk.desktop</id>
   <name>Anydesk</name>
```
Comment (wjt) [inline com.anydesk.Anydesk.metainfo.xml]:
```suggestion
  <name>AnyDesk</name>
```

File: com.anydesk.Anydesk.json [hunk 3]
```diff
@@ -154,7 +154,7 @@
                 },
                 {
                     "type": "file",
-                    "path": "com.anydesk.Anydesk.appdata.xml"
+                    "path": "com.anydesk.Anydesk.metainfo.xml"
```
Comment (wjt) [inline com.anydesk.Anydesk.json]: Why are you renaming this file, by the way?

**Comment by LeandroStanger** [comment]
bot, build com.anydesk.Anydesk

**Pull request closed without merging**